x-pack/plugins/cases/public/plugin.ts

```diff
@@ -51,6 +53,10 @@ export class CasesUiPlugin implements Plugin<void, CasesUiStart, SetupPlugins, S
       hooks: {
         getUseCasesAddToNewCaseFlyout: useCasesAddToNewCaseFlyout,
         getUseCasesAddToExistingCaseModal: useCasesAddToExistingCaseModal,
+        useCasesToast,
+      },
+      helpers: {
+        getRuleIdFromEvent,
```

Comment:
I know we discussed it together that now I am seeing the code I am a bit skeptical if we should put this logic to cases. Does the same logic apply to observability?

Reply:
yes it does.

Reply:
Hmm ok, let's keep it for the moment.